modules/manage/partials/tiered-storage.adoc

```diff
@@ -395,11 +471,115 @@ IMPORTANT: Do not set an object storage property to an empty string `""` or to `
 
 ==== Microsoft ABS/ADLS
 
-NOTE: Redpanda supports storage accounts configured for ADLS Gen2 with https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-namespace[hierarchical namespaces^] enabled. For hierarchical namespaces created with a custom endpoint, set the properties `cloud_storage_azure_adls_endpoint` and `cloud_storage_azure_adls_port` in the Helm chart through the command line (`--set storage.tiered.config`) or the YAML file. If you haven't configured custom endpoints in Azure, there's no need to edit these properties.
+You can configure access to Azure Blob Storage with either shared keys or Azure's managed identities system to securely interact with Azure Blob Storage. Shared keys, as static credentials, are simple to use but require manual management and vigilant security practices to prevent breaches due to their unchanging nature. In contrast, managed identities provide a more secure and maintenance-free solution by automating credential management and rotation, though they are exclusive to the Azure ecosystem.
+
+include::manage:partial$azure-blob-limitations.adoc[]
+
+AKS supports OpenID Connect (OIDC) federation. When a cluster is set and registered as an OIDC provider, you can configure Redpanda to use this federated identity to securely access Azure Blob Storage without needing to manage static credentials.
+
+To configure access to an Azure container with a managed identity in AKS:
+
+. Configure an xref:manage:security/iam-roles.adoc#configuring-iam-roles[Azure managed identity].
+. Override the following required cluster properties in the Helm chart:
++
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    storage:
+      tiered:
+        config:
+          cloud_storage_enabled: true
+          cloud_storage_credentials_source: azure_aks_oidc_federation
+          cloud_storage_azure_managed_identity_id: <managed-identity-client-id>
```

Comment:
`cloud_storage_azure_managed_identity_id` 
should not go with `cloud_storage_credentials_source: azure_aks_oidc_federation`

it's only relevant for 
`cloud_storage_credentials_source: azure_vm_instance_metadata`

was this from the example provided by @c4milo ?

Reply:
Nop